ultralytics/utils/benchmarks.py

```diff
@@ -353,10 +356,18 @@ def profile(self):
                 model_info = model.info()
                 if self.trt and self.device.type != "cpu" and not engine_file.is_file():
                     engine_file = model.export(
-                        format="engine", half=self.half, imgsz=self.imgsz, device=self.device, verbose=False
+                        format="engine",
+                        half=self.half,
+                        imgsz=self.imgsz,
+                        device=self.device,
+                        verbose=False,
                     )
                 onnx_file = model.export(
-                    format="onnx", half=self.half, imgsz=self.imgsz, simplify=True, device=self.device, verbose=False
+                    format="onnx",
+                    imgsz=self.imgsz,
+                    simplify=True,
```

Comment:
@Laughing-q @glenn-jocher should we set simplify=True as default in default.yaml

Reply:
@inisis yes I suppose this can't hurt. Can you submit a PR to update the default and remove L368 here?